.github/workflows/ci.yml

```diff
@@ -89,43 +89,33 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04]
-        containerd: [v1.6.21, v1.7.1]
+        containerd: [1.6.21, 1.7.1]
 
     steps:
       - name: Checkout extensions
         uses: actions/checkout@v4
 
+      - name: Download containerd archive
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh release download v${{ matrix.containerd }} \
+            --repo containerd/containerd \
+            --pattern 'containerd-${{ matrix.containerd }}-linux-amd64.tar.gz' \
+            --output containerd.tar.gz
+
+      - name: Extract containerd binaries to $HOME/.local/bin
+        run: |
+          mkdir -p $HOME/.local/bin
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
```

Comment:
why not to `/usr/local/bin/` ? it's already in `PATH`

Reply:
I tried to originally but it looks like GitHub has some restrictions on this 
<img width="444" alt="image" src="https://github.com/containerd/rust-extensions/assets/10913471/004e6774-b04c-423a-84bf-be083cd71d9a">